kaverigond/HTML/html assign/kaveri_assign/Q4.html

```diff
@@ -0,0 +1,15 @@
+</DOCTYPE html>
+<html>
+    <head>
+        <title> My First HTML Page</title> 
+        <body>
+            <h1> Introduction </h1>
+            <p>
+            <h3>My name is Kaveri Gond</h3>
+            <h3>I am from Khamgaon</h3>
+            <h3>completed my BE</h3>
+            <h2> Thank You</h2>
+            </p>
+            
+        </body>
+    </head>
```

Comment:
wrong code 
head tag need to close before body tag
html closing tag is also missing